### PR TITLE
Move FramePointerValidatorServiceImpl to its own module

### PR DESCRIPTION
**Pull request by dpallotti**
Part of a series of changes that aim to isolate each gRPC service into its own
module for better readability and maintenance. Currently, the Service module
contains multiple independent subsets of classes contributing to the different
services, so we extract those.

Test: Run Orbit with `--enable_frame_pointer_validator` and verify that
"Verify Frame Pointers" context menu action works.

### CODE REVIEW

File: src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp [hunk 1]
```diff
@@ -12,17 +12,16 @@
 
 #include "FramePointerValidator/FramePointerValidator.h"
 #include "ObjectUtils/ElfFile.h"
-#include "OrbitBase/Result.h"
 #include "code_block.pb.h"
 
-namespace orbit_service {
+namespace orbit_frame_pointer_validator_service_impl {
```
Comment (dimitry-) [inline src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp:17]: maybe without _impl?

**Review by florian-kuebler — APPROVED** [review]
lgtm! thanks!

File: src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp (hunk 1 above)
Comment (dpallotti) [reply src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp:17]: Hmm, the previous change has `_impl` (`crash_service_impl`). I don't have an opinion either way, though. Would you like me to remove it from both?

Comment (dimitry-) [reply src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp:17]: yep, makes sense to drop impl for both, sorry I missed it.

Comment (dpallotti) [reply src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp:17]: Only the namespace, yes?

Comment (dimitry-) [reply src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp:17]: and module name

Comment (dpallotti) [reply src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp]: Done for FramePointerValidatorService.

**Review by dimitry- — APPROVED** [review]

**Pull request merged**